esp/esp/users/models/__init__.py

```diff
@@ -253,6 +253,11 @@ def getLastProfile(self):
         from esp.program.models import RegistrationProfile
         return RegistrationProfile.getLastProfile(self)
 
+    def get_last_program_with_profile(self):
+        # ditto
```

Comment:
nit: can you at least say what you're referencing?  it's easy for someone to add a new method in between and this will no longer make any sense.